includes/Core/REST_API/REST_Routes.php

```diff
@@ -93,6 +93,33 @@ function( $server ) {
 				$this->register_routes( $server );
 			}
 		);
+
+		add_filter(
+			'do_parse_request',
+			function( $do_parse_request, $wp ) {
+				add_filter(
+					'query_vars',
+					function( $vars ) use ( $wp ) {
```

Comment:
Why the nested filters? Can we not simply filter `query_vars` and use a check to see if it's a REST request?
E.g. `defined( 'REST_REQUEST' ) && REST_REQUEST`

Reply:
@aaemnnosttv we need to make sure that we turn off `orderby` only for our endpoints because otherwise we would break standard or 3rd party endpoints which is not what we need.

Reply:
I see what you mean now, it's only used to provide access to the `$wp` instance before the `query_vars` filter is run. I wasn't suggesting we would _only_ do it based on if it was a rest request or not and I think the way you have it now is fine, it's just a bit odd to see it used this way.